modules/virt-creating-template-wizard-web.adoc

```diff
@@ -10,27 +10,44 @@ The web console features an interactive wizard that guides you through the *Gene
 All required fields are marked with a `*`. The wizard prevents you from moving to the next step
 until you provide values in the required fields.
 
+[NOTE]
+====
+The interactive wizard guides you to create a custom virtual machine template where you specify the operating system, boot source, flavor, and other settings.
```

Comment:
I performed a global search and modified all .adoc files associated with virtual machine templates. The .adoc files associated with VMs - that change is planned to be made as part of CNV-6749 because that PR is focused only on VMs. You might also want to mention this to other writers that are working with modules where interactive wizard is mentioned. I didn't want to expand the scope of these changes to .adoc files that were not associated with Virtual Machine Templates and this PR.